src/lib/dialog/dialog.scss

```diff
@@ -27,34 +30,61 @@ $mat-dialog-button-margin: 8px !default;
   }
 }
 
+.mat-dialog-container-scrollable {
+  padding: 0;
+
+  // Since there are 5-6 levels of elements down before we can reach
+  // the projected content, we have to use a class that lets us propagate
+  // the dimensions down to the relevant flexboxes, in order for IE to
+  // work correctly.
+  &, .mat-dialog-component-host {
+    width: inherit;
+    min-width: inherit;
+    max-width: inherit;
+    height: inherit;
+    min-height: inherit;
+    max-height: inherit;
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+  }
+}
+
+.mat-dialog-title {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  flex-shrink: 0;
+  order: 1;
```

Comment:
Wouldn't specifying the `order` take away the ability to put custom content in between the regions?

Reply:
I suppose that it would, but the assumption is that if they're using the content element, they're putting all their content into it. I'm fine with removing it though, we haven't had it until now.

Reply:
If the only reason would be to let people put the dialog directives in any order, I'd rather omit it to give people control in case they need to do something special